src/components/Nav.js

```diff
@@ -1,11 +1,151 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.scss';
+import PRODUCTCATEGORYS from './productCategorys';
+import INTRODUCELUSH from './introduceLush';
```

Comment:
```suggestion
import PRODUCT_CATEGORIES from './productCategorys';
import INTRODUCE_LUSH from './introduceLush';
```
- categorys -> categories
- 언더스코어를 단어별로 끊어주는 게 읽기 좋아보입니다.